platform_versioned_docs/version-24.1/enterprise/configuration/configtables/data_features_yml.yml

```diff
@@ -0,0 +1,11 @@
+---
+-
+  tower.yml:            '`tower.content.max-file-size`'
+  Description: >
+    You can set the Data Explorer downloadable file size limit. Increasing this value may degrade Platform performance.
+  Value:                'Default: `25MB`'
+-
+  tower.yml:            '`tower.data.data-link-items.tree-list.max-allowed`'
+  Description: >
+    You can set the maximum number of items listed in Data Explorer.
```

Comment:
minor: I liked the above phrasing and would add here as well: `Increasing this value may degrade Platform performance.`